gateway/api/models.py

```diff
@@ -13,6 +15,28 @@ def get_upload_path(instance, filename):
     return f"{instance.author.username}/{instance.id}/{filename}"
 
 
+class JobConfig(models.Model):
+    """Job Configuration model."""
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    created = models.DateTimeField(auto_now_add=True)
+
+    auto_scaling = models.BooleanField(default=False, null=True)
+    workers = models.IntegerField(
+        null=True,
+    )
+    min_workers = models.IntegerField(
+        null=True,
+    )
+    max_workers = models.IntegerField(
+        null=True,
+    )
+    python_version = models.TextField(null=True, blank=True)
```

Comment:
it's better to have `ChoiceField` with hardcoded versions. It will enable validation + migration out of the box. 

https://docs.djangoproject.com/en/4.2/ref/models/fields/#django.db.models.Field.choices